pkg/scheduler/scheduler.go

```diff
@@ -381,14 +387,14 @@ func (s *Scheduler) doScheduleBinding(namespace, name string) (err error) {
 	if appliedPlacement := util.GetLabelValue(rb.Annotations, util.PolicyPlacementAnnotation); policyPlacementStr != appliedPlacement {
 		// policy placement changed, need schedule
 		klog.Infof("Start to schedule ResourceBinding(%s/%s) as placement changed", namespace, name)
-		err = s.scheduleResourceBinding(rb)
+		err = s.scheduleResourceBinding(rb, policyPlacement, policyPlacementStr)
```

Comment:
I wonder why you pass the two parameters related to placement into this function.

Reply:
Why do I have to calculate it again when I already got them? It seems unreasonable in the old version, but we try to reduce useless code as much as possible.

Reply:
If so, is the `getPlacement` function no longer needed?

Reply:
> If so, is the `getPlacement` function no longer needed?

I mean it is not necessary to call the function `getPlacement` twice.

Reply:
Got it, it's kind of a refactor.